easybuild/easyconfigs/j/jax/jax-0.4.24-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,124 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+# Updated by: Alex Domingo (Vrije Universiteit Brussel)
+# Updated by: Thomas Hoffmann (EMBL Heidelberg)
+easyblock = 'PythonBundle'
+
+name = 'jax'
+version = '0.4.24'
```

Comment:
@ThomasHoffmann77 There's a `jax` 0.4.25 release now, can we try updating to that and see if we're still seeing failing tests?

Reply:
@boegel I'll give 0.4.25 a try today

Reply:
I need this, so happy to review once you have updated.

Reply:
> I need this, so happy to review once you have updated.

@verdurin see #20119. (My local test runs did not finsh yet)